apps/system/utils/kdbg_date.c

```diff
@@ -205,6 +228,51 @@ static inline int date_settime(int argc, char **args)
 	}
 	tm.tm_year = (int)result - 1900;
 
+#if defined(CONFIG_LIBC_LOCALTIME) || defined(CONFIG_TIME_EXTENDED)
+	/* Get the day of the week.  NOTE: Accepts day-of-week from sunday to saturday */
+
+	token = args[6];
+	if (token == NULL) {
+		goto errout_bad_parm;
+	}
+
+	tm.tm_wday = day_of_week(token);
+        if (tm.tm_wday < 0) {
+		goto errout_bad_parm;
+	}
+
+	/* Get the day of the year.  NOTE: Accepts day-of-year from 0 to 365 */
+
+	token = args[7];
+	if (token == NULL) {
+		tm.tm_yday = 0;
+	}
+	else {
```

Comment:
Put this line at above (same line with closing bracket of if). This is a violation of coding rule.